playbooks/templates/standalone_parameters.yaml.j2

```diff
@@ -141,8 +141,10 @@ parameter_defaults:
   GlanceBackendID: central
   GlanceEnabledImportMethods: web-download,copy-image
   GlanceStoreDescription: 'central rbd glance store'
-{% endif %}
+{% else %}
+  # BZ#1901143 and BZ#1903005
```

Comment:
🙏

Every time a developer leaves a comment containing bug link next to a hacky workaround, a fairy gets its wings.